spec/shoryuken/cli_spec.rb

```diff
@@ -22,6 +22,12 @@
       allow(launcher).to receive(:stop)
     end
 
+    # reset shoryuken options because class variable hold value which is used previous test case
```

Comment:
When I run all specs, a spec(https://github.com/phstc/shoryuken/pull/261/files#diff-c4306b956d835c4acc8f27cd5f44f968R41) was failed.
Because class variable hold previous option value which is used previoust test case: https://github.com/phstc/shoryuken/pull/261/files#diff-c4306b956d835c4acc8f27cd5f44f968R35

It can't be helped as long as class variable use for Shoryuken.options.